Casks/c/crystaldiffract.rb

```diff
@@ -0,0 +1,31 @@
+cask "crystaldiffract" do
+  version "7.0.1"
+  sha256 :no_check
+
+  url "https://crystalmaker.com/downloads/crystaldiffract#{version.major}_mac.zip"
+  name "CrystalDiffract"
+  desc "Powder diffraction software including phase ID & rietveld refinement"
+  homepage "https://crystalmaker.com/crystaldiffract/index.html"
+
+  livecheck do
+    url "https://crystalmaker.com/support/updates/VersionData-CDM.xml"
+    regex(/\b(\d+\.\d+\.\d+)\b/)
+    strategy :xml do |xml, regex|
+      xml.get_elements("versiondata/versionlist/version").map { |item| item.attributes["number"][regex, 1] }
```

Comment:
```suggestion
    strategy :xml do |xml|
      xml.get_elements("//version").map { |item| item.attributes["number"] }
```

Same as https://github.com/Homebrew/homebrew-cask/pull/172222#discussion_r1581568335.